proto/scene.proto

```diff
@@ -58,19 +85,52 @@ Can be overridden in renderer
 message RenderOptions {
   // Camera from which scene should be rendered
   int32 camera_id = 1;
+  // Width of final image
   int32 width = 2;
+  // Height of final image
   int32 height = 3;
 }
 
-// Representation of cube 
-message Cube {}
+/*
+Representation of cube
+To determine the position of a cube look at Transform.position
+Transform.scale should be ignored
+If your renderer supports only axis aligned box Transform.rotation should be ignored
+For those who dare to support oriented bounding box here link how obb is implemented in opengl:
+http://www.opengl-tutorial.org/miscellaneous/clicking-on-objects/picking-with-custom-ray-obb-function/
```

Comment:
Just a reminder that HTML formatting works, so you can use `<a href="...">` to make links clickable in the doc.